homeassistant/components/proxmoxve/__init__.py

```diff
@@ -18,6 +19,9 @@
 
 _LOGGER = logging.getLogger(__name__)
 
+# Disable constant logs of requests to Proxmox
+logging.getLogger("proxmoxer.core").setLevel(level=logging.WARNING)
```

Comment:
This should preferably be fixed in the library instead. Please make a PR to the library lowering the logging level to debug for the concerned cases.

Reply:
It seems a new repo was recently created to continue development from the abandoned upstream repo.

https://github.com/proxmoxer/proxmoxer

Hope they can publish on pypi too.

Reply:
Yeah, I could open a PR, but it won't get fixed any time soon in the upstream.
I've reached out to the original maintainer @swayf to see if he will switch the pypi over to the organization's repo.

Reply:
sure we can do it.. I have already written the answer, please give me the account names in pypi, I will add you in pypi to maintain tha package. hopefully I will join you later. but right now I have no time for that =(

Reply:
fixing in upstream. I'll push out a new version of proxmoxer when the upstream is resolved and update the version used here

Reply:
Merged ;)

Reply:
version requirement updated with fixed version